package/src/utils/utils.ts

```diff
@@ -245,23 +243,22 @@ export type TriggerSettings<
   Us extends UnknownType = DefaultUserType,
 > = {
   '/'?: {
```

Comment:
The symbols used to trigger a suggestion list are hard coded both here, and in `const ACITriggerSettings` - can we have an enum or object or something to gather that in one place? Or somehow type it so typescript would tell us what the different options are? (I think the latter is trickier to do, though)